### DISCUSSION TITLE
Awesome console apps in C#

### DISCUSSION THREAD

**Discussion by giscus** [piraces/blog#701] [Announcements]
# Awesome console apps in C#

No more raw and boring console apps! Learn how to create fancy and useful console apps in C# with libraries made for this purpose. Meet the awesome libraries Spectre.Console and gui.cs.

https://piraces.dev/posts/awesome-console-apps-csharp/

**Comment by Cristiangonzelz** [comment]
git clone https://github.com/h4ck3r0/Metasploit-termux
